doc/releases/release-notes-3.3.rst

```diff
@@ -262,6 +262,33 @@ Architectures
 
 * RISC-V
 
+  * Converted :kconfig:option:`CONFIG_MP_MAX_CPUS` to
+    :kconfig:option:`CONFIG_MP_MAX_CPUS`.
```

Comment:
Is that correct?

Reply:
Thanks for pointing that out. I'll create a PR correcting this soon.

Reply:
Done in this PR: https://github.com/zephyrproject-rtos/zephyr/pull/54808.